### DISCUSSION TITLE
Улучшение UI/UX поля с файлами

### DISCUSSION THREAD

**Discussion by diSWAG** [totumonline/totum-ru-issues-and-discussions#361] [Как это сделать?] [answered]
Текущая логика вызова меню редактирования списка прикрепленных файлов построена через правую кнопку.
<img width="448" alt="Снимок экрана 2023-04-28 в 12 46 59" src="https://user-images.githubusercontent.com/122656882/235115441-787db2c7-767b-4883-81b9-85cd2edca71c.png">

Но очень часто натыкаюсь, на то что удобнее и быстрее добавить файл к существующим было бы, если в настройках был бы чекбокс активирующий, нечто подобное:
<img width="120" alt="Снимок экрана 2023-04-28 в 12 55 28" src="https://user-images.githubusercontent.com/122656882/235117775-18ad6a00-7c6e-434e-9ce3-1b0a9b53c651.png">

**Comment by ttm-alexeypolunin** [comment, marked as answer]
В коде форматирования поля

```
f1=: setFormat(editbutton: true)
```

**Reply by diSWAG** [reply]
Супер! Спасибо!
